pyproject.toml

```diff
@@ -124,9 +100,54 @@ strict_equality = true
 exclude = ['py4ai/core/_version\.py$']
 
 [[tool.mypy.overrides]]
+ignore_missing_imports = true
 module = [
-    "pandas.*", "sqlalchemy.*", "matplotlib.*", "sklearn.*", "seaborn.*", "scipy.*", "dill.*", "cfg_load.*",
-    "bson.*", "pymongo.*", "motor.*", "mongomock_motor.*"
+    "bson.*",
+    "cfg_load.*",
+    "dill.*",
+    "matplotlib.*",
+    "mongomock_motor.*",
+    "motor.*",
+    "pandas.*",
+    "pymongo.*",
+    "scipy.*",
+    "seaborn.*",
+    "sklearn.*",
+    "sqlalchemy.*",
 ]
```

Comment:
That is not specific to this PR but I think that some of these modules can be removed from this list: some of these packages are not in the dependencies of this library and others (like pandas) had been statically typed in the meantime.

Reply:
Ack. But I would leave the cleaning outside of this PR, as it does require some work and thoughts (maybe some modules are also not needed anymore)